packages/dotlottie-js/src/node/dotlottie.ts

```diff
@@ -118,17 +117,25 @@ export class DotLottie extends DotLottieCommon {
 
       const tmpImages = [];
 
+      if (contentObj['manifest.json'] === undefined) {
+        throw createError('manifest.json is missing from the dotLottie file');
+      }
+
+      // Parse the manifest first so that we can pick up animation settings
+      const manifest = JSON.parse(strFromU8(contentObj['manifest.json'] as Uint8Array, false));
+      const { author, custom, description, generator, keywords, version } = manifest;
+
+      dotlottie.setAuthor(author as string);
+      dotlottie.setCustomData(custom);
+      dotlottie.setDescription(description as string);
+      dotlottie.setGenerator(generator as string);
+      dotlottie.setKeywords(keywords);
+      dotlottie.setVersion(version as string);
```

Comment:
what if `author`, `custom` , `description`, ...etc are undefined ? 
shall we have a runtime validation if it's of type `string` ?

Reply:
What if I re-wrote setAuthor etc.. like this ?

```
  public setAuthor(author: string | undefined): DotLottieCommon {
    this._author = author ?? 'LottieFiles';

    return this;
  }
```

So that the check is inside common code, and I dont have to re-write checks for node + web ?

I'll add runtime check inside dotlottie-common so that its shared as well